src/Client/Web/Bit.Client.Web.BlazorUI/Components/Pivot/BitPivotItem.razor

```diff
@@ -0,0 +1,12 @@
+﻿@namespace Bit.Client.Web.BlazorUI
+@inherits BitComponentBase
+
+<div @ref="RootElement"
+     style="@StyleBuilder.Value"
+     class="@ClassBuilder.Value">
+
+    @if (Pivot.HeadersOnly is false && Pivot?.Items[Pivot.SelectedKey] == this && this.Visibility != ComponentVisibility.Hidden)
```

Comment:
inconsistent Pivot null check:
Pivot.HeadersOnly    vs    Pivot?.Items

Reply:
used null check for Pivot.HeadersOnly